debezium-connector-sqlserver/src/test/java/io/debezium/connector/sqlserver/SqlServerConnectorIT.java

```diff
@@ -2569,6 +2572,100 @@ public void shouldIncludeDatabaseNameIntoTopicAndSchemaNamesInMultiPartitionMode
         assertThat(record.valueSchema().name()).isEqualTo("server1.testDB.dbo.tablea.Envelope");
     }
 
+    @Test
+    @FixFor("DBZ-2793")
+    public void shouldApplySchemaFilters() throws Exception {
+        connection.execute("DROP SCHEMA IF EXISTS s1");
```

Comment:
@vjuranek I think this will fail if the schem is non empty.
How about reordering the commands? delete all tables, schemas and then create all schemas and tables?

Reply:
Hm, it worked for me, but maybe the during each test run new container was started. I re-ordered the commands and double check that it works when run twice (and no other container is started during the test), so hopefully should be fine now.

Reply:
@vjuranek Thanks for fixing. The issue would be visible if a previous test execution would not clean up the schema - either due to test failure or bug in test which would spread here.